docs/en/samples/image_pull_secrets.md

```diff
@@ -0,0 +1,71 @@
+# image pull secrets
+
+
+In private scenario, maybe you need to set image pull secret.
```

Comment:
If the images of fluid runtime is in private docker registry,  it's necessary to set image pull secret.

Reply:
Thank you for your suggestion, done